client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/logging/DefaultLogUploadStrategy.java

```diff
@@ -26,32 +26,39 @@
 public class DefaultLogUploadStrategy implements LogUploadStrategy {
     private static final Logger LOG = LoggerFactory.getLogger(DefaultLogUploadStrategy.class);
 
-    private static final int DEFAULT_UPLOAD_TIMEOUT = 2 * 60;
-    private static final int DEFAULT_UPLOAD_CHECK_PERIOD = 30;
-    private static final int DEFAULT_RETRY_PERIOD = 5 * 60;
-    private static final int DEFAULT_UPLOAD_VOLUME_THRESHOLD = 8 * 1024;
-    private static final int DEFAULT_UPLOAD_COUNT_THRESHOLD = 64;
-    private static final int DEFAULT_BATCH_SIZE = 8 * 1024;
-    private static final int DEFAULT_BATCH_COUNT = 256;
-
-    private int timeout = DEFAULT_UPLOAD_TIMEOUT;
-    private int uploadCheckPeriod = DEFAULT_UPLOAD_CHECK_PERIOD;
-    private int retryPeriod = DEFAULT_RETRY_PERIOD;
-    private int volumeThreshold = DEFAULT_UPLOAD_VOLUME_THRESHOLD;
-    private int countThreshold = DEFAULT_UPLOAD_COUNT_THRESHOLD;
-    private int batchSize = DEFAULT_BATCH_SIZE;
-    private int batchCount = DEFAULT_BATCH_COUNT;
+    protected static final int DEFAULT_UPLOAD_TIMEOUT = 2 * 60;
```

Comment:
Ex: https://github.com/kaaproject/kaa/pull/287/files#diff-dec987a77bcc430ae57b913b1c26dfcaR23
or you mean all constant variables? protected scope gives ability to use this defaults in user custom inerhited classes.